include/z64math.h

```diff
@@ -103,4 +103,12 @@ typedef struct {
 #define DEG_TO_RAD(degrees) ((degrees) * (M_PI / 180.0f))
 #define BINANG_TO_DEGF(binang) ((f32)binang * (360.0001525f / 65535.0f))
 
+/**
+ * Vector macros
+ */
```

Comment:
we can do a // comment to take up less space, same for the one thats already there. didnt notice it before